docs/changelog/4.5.rst

```diff
@@ -1,6 +1,36 @@
 Changelogs for 4.5.x
 ====================
 
+.. changelog::
+  :version: 4.5.0
+  :released: 13th of July 2021
+
+  This is release 4.5.0 of the Authoritative Server.
+  This release contains a ton of improvements and bug fixes compared to 4.4, but very few user visible changes.
+
+  There are two notable new features:
+
+  * The 'zone cache', which allows PowerDNS to keep a list of zones in memory, updated periodically.
+    With this cache, PowerDNS can avoid hitting the database with queries for unknown domains.
+    In some setups, and some attack scenarios, this can make a serious performance difference.
```

Comment:
"Users of backends with dynamically generated zones want to disable this / read the upgrade notes extremely carefully"?